File: NetSix_java/OffertaAbbonati.java
public class OffertaAbbonati{
    public static String[] serietv = {"CSI", "Game of thrones", "Friends", "Breaking bad", "Mr Robot", "Jack ryan"};
    public static int[] episodi = {140, 80, 200, 80, 45, 40};

    public static int getDisponibilita(String titolo, int episodio){
        for(int i = 0; i < 6; i++)
            if(serietv[i].equals(titolo)){
                if(episodio <= episodi[i])
                    return 1;
                else
                    return 0;
            }
        return -1;
    }
}

File: NetSix_C/OffertaAbbonati.c
#include "OffertaAbbonati.h"
#include <string.h>
#include <stdio.h>

char* serietv[6] = {"Friends", "How i met your mother", "Game of thrones", "Breaking bad", "CSI", "The boys"};
int episodi[6] = {52, 48, 50, 120, 250, 20};

int getDisponibilita(char* titolo, int episodio){
    for(int i = 0; i < 6; i++){
        if(strncmp(serietv[i], titolo, strlen(serietv[i])) == 0){
            if(episodio <= episodi[i])
                return 1;
            else
                return 0;
        }
    }

    return -1;
}